clang/test/CIR/CodeGen/vectype.cpp

```diff
@@ -0,0 +1,15 @@
+// RUN: %clang_cc1 -std=c++17 -triple x86_64-unknown-linux-gnu -fclangir-enable -emit-cir %s -o - | FileCheck %s
+
+typedef int int4 __attribute__((vector_size(16)));
+int main() {
+  int4 a = { 1, 2, 3, 4 };
+  int4 b = { 5, 6, 7, 8 };
```

Comment:
Since `cir.vec` accepts values coming from other things than cir.const, might be good to add a another function to test that. Also, rename functions with something other than `main` to avoid confusion

Reply:
Done.